draft-ietf-quic-transport.md

```diff
@@ -489,27 +431,30 @@ which can allow for better stream prioritization.
 
```

Comment:
Perhaps a note here on loose coupling between application stream handles and QUIC internal stream ID's?